muy_map_conf.js

```diff
@@ -28,20 +31,27 @@ function init(config) {
           lng: parseFloat(place.location.lng, 10)
         }
       });
+      mapMarkers.push(marker);
   
-      var content = '<h2>' + place.title + '</h2>' + place.address; 
       var infoWindow = new google.maps.InfoWindow({
-        content: content
+        content: `<h2>${place.title}</h2>${place.content}`
```

Comment:
String interpolation won't work in any version of IE.

Reply:
😱 I guess i've finally been infected by living in <current_year>. I'll fix it lol